bin/deploy-vhost

```diff
@@ -236,7 +238,11 @@ sub nginx_graceful {
 
 sub graceful {
     # Prompt webserver to reread files, on machines which have apache...
-    apache_graceful();
+    if (!$conf->{touch_to_restart} || $restart_apache) {
```

Comment:
Ah I see, I was missing the point - thinking that it was being used to control whether or not to restart apache in some cases where there wasn't a `touch_to_restart` method rather than ensuring that there is a way of restarting apache when necessary when there is.